server/utils/db/init.sql

```diff
@@ -22,6 +22,13 @@ CREATE table IF NOT EXISTS babysitters (
     comments TEXT
 );
 
+INSERT INTO babysitters (babysitter_name, city, street, experience, age, phone_number, working_hours, image_string, comments)
+VALUES 
+('Amit', 'Holon', 'Arlozorov', 'mid', 20, '0505050050', '{"monday": "9am-3pm", "tuesday": "9am-3pm", "wednesday": "9am-3pm", "thursday": "9am-3pm", "friday": "9am-3pm"}' , 'https://media.licdn.com/dms/image/C4D03AQFREc4ImmXDAw/profile-displayphoto-shrink_100_100/0/1650354859697?e=1719446400&v=beta&t=aAcbM2Tgyq_7UkkFytyNuxtqIejm1XefENtIuNO2BIo', 'I need a job, please hire me.'),
```

Comment:
image_string is not supposed to be initialized  yet, and when it will initialized it will be set from another service and not hard coded in. Please remove the URLs and leave the field empty (or add a placeholder string if needed)

Reply:
done